src/components/ProgramRecordSendModal/data/service.js

```diff
@@ -0,0 +1,16 @@
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
+import { logError } from '@edx/frontend-platform/logging';
+
+async function sendRecords(programId, username, orgId) {
+  try {
+    const url = `${getConfig().CREDENTIALS_BASE_URL}/records/programs/${programId}/send`;
+    const response = await getAuthenticatedHttpClient().post(url, { username, pathway_id: orgId });
```

Comment:
Is this correct? Has this been tested? It looks like we're just sending the username as text and I don't think this will be valid data.

If I remember correctly, the endpoint is expecting data in the form of:
```
{
    "username": "<username>",
    "pathway_id": orgId
}
```
See the [ProgramSendView] class in the Credentials backend.(https://github.com/openedx/credentials/blob/dc5f397fbc67ad2b2304b1dd91a9879f35f22bb2/credentials/apps/records/views.py#L201)

Reply:
I believe this works as intended. I was under the impression that
 `{ username, 
 pathway_id: orgId }` 
 is equal to 
`{ username: username, pathway_id: orgId }`

Reply:
Believe or know it works as intended? It sounds like we may not be sure? Can we verify that it does work?

Reply:
That is how Javascript behaves. It's a little bit of syntactical sugar

Reply:
Yes it still works. It will however error the second time you send the record because the API checks for that

Reply:
Thanks for the clarification!

Reply:
@justinhynes heres the shorthand linting rule (which I think we enforce) for reference: https://eslint.org/docs/latest/rules/object-shorthand